### PR TITLE
Contact me form

### PR DESCRIPTION
**Pull request by Goldinium**
In this pull request we implemented 

- [ ] forms
- [ ] sending button
- [ ] background images
- [ ] sending feature through Formespree service
- [ ] limitation on maximum characters that can be typed
- [ ] the form can't be sent if some or all the fields are empty

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 > Portoflio Mobile Website
```
Comment (DanteAlonsoHT) [inline README.md:6]: Please, I know you can improve the description for your project, try to highlight the technologies and techniques you used, what is the main scope for this project, and so on.

- [ ] Improve project description in README file.

File: README.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ https://github.com/Goldinium/Portfolio_works.git
 
 - GitHub: [@goldinium](https://github.com/goldinium)
 
+👤 **Author2**
```
Comment (DanteAlonsoHT) [inline README.md:37]: Instead: `👤 **Author2**`
You can use `👤 **Your Name**`
In order to improve how your README file looks like.

- [ ] [OPTIONAL] Add your name in Author Section.

File: index.html [hunk 3]
```diff
@@ -12,135 +12,30 @@
...
-                    <li class="javascript"><img class="langpic" alt="js" src=img/js.png>Javascript</li>
-                    <li class="html1"><img class="langpic" alt="html" src=img/html.png>HTML5</li>
-                    <li class="css1"><img class="langpic" alt="css" src=img/css.png>CSS3</li>
+                    <li><input type="text" id="name" class="name" maxlength="30" required placeholder="Your name">
+                    </li>
+                    <li><input type="email" id="email" class="email" required placeholder="Your email"></li>
+                    <li><textarea name="message" id="message" class="message" maxlength="500"
+                            placeholder="Write your message here" required></textarea></li>
+                    
                 </ul>
```
Comment (DanteAlonsoHT) [inline index.html:34]: You can remove the list `ul` with `li` elements from your form due to is not a semantic HTML.

- [ ] [OPTIONAL] Remove `ul` list with `li` elements.

File: style.css [hunk 4]
```diff
@@ -1,199 +1,104 @@
...
+  text-align: center;
 }
 
-.project_button:hover {
-  color: rgb(138, 4, 248);
+.button:hover {
+  box-shadow: 0 8px rgba(64, 83, 252, 0.24);
+  background-color: #6070ff;
+  color: #fff;
 }
```
Comment (DanteAlonsoHT) [inline style.css:85]: I recommend you add the attribute `cursor: pointer;` in order to improve the UX for users 😉 

- [ ] [OPTIONAL] Add pointer cursor when someone hovers the buttons.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Goldinium and @Fikerte-T, 

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- The rest of your project is missing, I expect you will add it to your portfolio after merging it.

<hr>

I know you have the skills enough to finish these changes required in time 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by barackm — APPROVED** [review]
Hi @Goldinium, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please let us keep our commit messages consistent, we should always start to commit messages with uppercase letters and they should always be in the present tense. Let's make better mistakes tomorrow! 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**